frontend/admin/src/js/components/searchRequest/SingleSearchRequestTable.tsx

```diff
@@ -123,22 +123,28 @@ export const SingleSearchRequestTable: React.FunctionComponent<
           description:
             "Title displayed on the single search request table operational requirements column.",
         }),
-        accessor: ({ acceptedOperationalRequirements }) =>
-          acceptedOperationalRequirements?.map((operationalRequirement) => {
-            return (
-              <Pill key={operationalRequirement} color="primary" mode="outline">
-                {intl.formatMessage(
-                  operationalRequirement
-                    ? getOperationalRequirement(operationalRequirement)
-                    : {
-                        defaultMessage: "Error: Name not found.",
-                        description:
-                          "Error message displayed on the single search request table operational requirements column.",
-                      },
-                )}
-              </Pill>
-            );
-          }),
+        accessor: ({ user }) =>
+          user.acceptedOperationalRequirements?.map(
```

Comment:
Is `user` guaranteed to be non-null? Or should this be `user?.acceptedOperationalRequirements?.map(`?

Reply:
I took a closer look, you were right, typing guarantees it to be non-null, my mistake. I guess my env wasn't fully built before. In which case, its better to leave off the `?.` if possible.

Reply:
sure, can revert it
https://github.com/GCTC-NTGC/gc-digital-talent/pull/5055/commits/367c943da677cecdc39a3f91acf56fd0e4ea134b